rita/src/rita_common/dashboard/wallet.rs

```diff
@@ -0,0 +1,55 @@
+use super::*;
+
+pub fn withdraw(path: Path<(Address, u64)>) -> Box<Future<Item = HttpResponse, Error = Error>> {
+    let address = path.0;
+    let amount = path.1;
+    debug!("/withdraw/{:#x}/{} hit", address, amount);
+
+    let full_node = get_web3_server();
+    let web3 = Web3Client::new(&full_node);
+    let payment_settings = SETTING.get_payment();
+
+    let tx = Transaction {
+        nonce: payment_settings.nonce.clone(),
+        gas_price: payment_settings.gas_price.clone(),
+        gas_limit: "21000".parse().unwrap(),
+        to: address,
+        value: amount.into(),
+        data: Vec::new(),
+        signature: None,
+    };
+    // TODO figure out the whole network id thing
+    let transaction_signed = tx.sign(
+        &SETTING
+            .get_payment()
+            .eth_private_key
+            .expect("No private key configured!"),
+        None,
```

Comment:
Its quite important especially for testnets where they have different network IDs and its as simple as querying `web3.net_version()` on bootup, or before signing each transaction.